src/MyEventEmitter.js

```diff
@@ -1,12 +1,122 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  validateEventName(eventName) {
+    if (typeof eventName !== 'string') {
+      throw new Error('The event name must be a string');
+    }
+  }
+
+  validateListener(listener) {
+    if (typeof listener !== 'function') {
+      throw new Error('The listener must be a fanction');
+    }
+  }
```

Comment:
you could combine the two to save some codelines
```suggestion
  validateListener(eventName) {
    if (typeof eventName !== 'string') {
      throw new Error('The event name must be a string');
    }
    
    if (typeof listener !== 'function') {
      throw new Error('The listener must be a fanction');
    }
  }
```